.circleci/integration-tests/master_dag.py

```diff
@@ -93,8 +93,7 @@ def prepare_dag_dependency(task_info, execution_time):
         {"s3_sensor_dag": "example_s3_sensor"},
         {"redshift_sql_dag": "example_async_redshift_sql"},
         {"redshift_cluster_mgmt_dag": "example_async_redshift_cluster_management"},
-        {"emr_job_flow_sensor_dag": "example_emr_job_flow_sensor"},
-        {"emr_cluster_dag": "example_emr_step_sensor"},
+        {"example_emr_sensor_dag": "example_emr_sensor"},
```

Comment:
```suggestion
        {"emr_sensor_dag": "example_emr_sensor"},
```

Should the key be renamed to the above to keep it consistent with other key names?

Reply:
Yeah, Fixed